Makefile.am

```diff
@@ -90,3 +90,11 @@ test_test_scrypt_SOURCES =	test/test_scrypt.c			\
 			lib/crypto/crypto_scrypt_smix.c			\
 			cpusupport-config.h
 test_test_scrypt_LDADD=	libscrypt_sse2.a
+
+.PHONY: test
+# we can't only build "scrypt test/test_scrypt" because that won't build
+# the BUILT_SOURCES.
```

Comment:
`make scrypt` seems to work fine for me?  And `cpusupport-config.h` (which I assume is the file you meant) is listed in `scrypt_SOURCES` so unless autotools is even more broken than I thought it should be fine?